website/source/api/discovery-chain.html.md

```diff
@@ -0,0 +1,541 @@
+---
+layout: api
+page_title: Discovery Chain - HTTP API
+sidebar_current: api-discovery-chain
+description: |-
+  The /discovery-chain endpoints are for interacting with the discovery chain.
+---
+
+# Discovery Chain HTTP Endpoint <sup>beta</sup>
+
+The `/discovery-chain` endpoint returns the compiled [discovery
+chain](/docs/internals/discovery-chain.html) for a service.
+
+This will fetch all related [configuration
+entries](/docs/agent/config_entries.html) and then normalize and distill them
+down into a concrete, actionable format for use by a [connect
+proxy](/docs/connect/proxies.html) implementation. This is a key component of
+[L7 Traffic Management](/docs/connect/l7-traffic-management.html).
+
+~> This is a low-level API primarily targeted at external connect proxy
+implementations.
+
+## Read Compiled Discovery Chain
```

Comment:
I'm not sure I've seen us combine methods before. What do you think of having a section for GET (Read Compiled Discovery Chain) and POST (Update the Discovery Chain). 

Also, why are we saying override instead of update or change?

Reply:
All of these endpoints are read-only (GET and POST). GET requests cannot have request bodies, so if any of the override parameters are required rather than having an increasingly longer list of parameters on the URL those are all encoded as a request body and sent over a method that allows bodies.